### ISSUE TITLE
Field maintenance_event_type in vw_qgep_maintenance is redundant

### ISSUE THREAD

**Issue by varrieta** [QGEP/QGEP#629]
Is there a reason for the field `maintenance_event_type` to  exist ? It seems redundant because the field `kind` could be use as well:

Maintenance_event_type = 'examination' if kind = 4564
Maintenance_event_type = 'maintenance_event' if kind <> 4564

![image](https://user-images.githubusercontent.com/20111558/106590264-3326cc80-654d-11eb-815c-ba0baafafd0d.png)

**Comment by varrieta** [comment]
@sjib Do you know why the view is made that way?

**Comment by sjib** [comment]
@varrieta No. Does maintenance_event_type has another value list than kind? Do you see who originally created that view?

**Comment by varrieta** [comment]
Yes maintenance_event has two value list:  'kind' and 'status'.

No I can't see who created it

**Comment by sjib** [comment]
#626 - these two issues might be connected

**Comment by m-kuhn** [comment]
I am not sure where this comes from, I cannot find any reference in this organisation https://github.com/search?q=org%3AQGEP+maintenance_event_type or the git history of qgep datamodel. Is this also in the demo data?
